README.md

```diff
@@ -194,15 +259,15 @@ this README. However, a few details about how the job is configurd are provided
   The `yoyo-migrations` library will obtain this value by using the 
   [`getpass.getuser` function](https://docs.python.org/3/library/getpass.html) from the Python standard library.
 
-  With the above two variables set, the `env` block in the `.yaml` will look something like this:
+  With the above two variables set, the `env` block in the `.yml` will look something like this:
 
     ```yaml
-  - env:
-      - name: ENV_FILE
-        value: /config/secrets/env_test.json
-      - name: USER
-        value: project_name
-  ```
+    - env:
+        - name: ENV_FILE
+            value: /config/secrets/env_test.json
+        - name: USER
+            value: project_name
```

Comment:
Is this standard indentation for YAML?

Reply:
I'll check.  Codacy didn't complain.

Reply:
Maybe not.

One source (the search result highlighted by Google) recommends indentation of 2 spaces, not 4.  However, [they also say](https://docs.saltstack.com/en/master/topics/troubleshooting/yaml_idiosyncrasies.html), "but YAML will follow whatever indentation system that the individual file uses."

Furthermore, the [YAML 1.2 specifications](https://yaml.org/spec/) don't give a specific number of spaces, but say that it needs to be consistent within a single file.  Their examples show 2, 4, and sometimes even 3 space indentation.

My Markdown editor is set to 4 because that's what that markup expects.  That's the problem we get when embedding code in text.  A different editor might not have this problem, I suppose.

Since you seem to be concerned by 4-space indentation, I'll change the indentation of this code to 2 spaces (within the Markdown quotes) this time just to get this PR though review and merged.

Reply:
I mostly didn't know why you were indenting those lines more than the `-` lines. I'm really not very experienced with YAML, so I'm not going to make a big deal about this.

Reply:
I think we're both making nitpick-y review comments that need to be prefixed with "nit:" as suggested by the ["How to Make Good Code Reviews Better"](https://stackoverflow.blog/2019/09/30/how-to-make-good-code-reviews-better/) article @jonespm recommended recently.

Reply:
Yeah, I've started using "Nitpick:". But you can assume that, if I've already approved something and haven't gone back and requested changes, that you can merge without resolving any open questions.